pahfit/features/features.py

```diff
@@ -137,7 +137,7 @@ class Features(Table):
     _group_attrs = set(('bounds', 'features', 'kind'))  # group-level attributes
     _param_attrs = set(('value', 'bounds'))  # Each parameter can have these attributes
     _no_bounds = set(('name', 'group', 'kind', 'geometry', 'model'))  # str attributes (no bounds)
-    _bounds_dtype = np.dtype([("val", "f4"), ("min", "f4"), ("max", "f4")]) # bounded param type
+    _bounds_dtype = np.dtype([("val", float), ("min", float), ("max", float)])  # bounded param type
```

Comment:
This reminds me we need to test `f4` vs. `f8` at some point.  I suppose that could be a fitter issue, whether it wants to use doubles in its fit (I suspect my approach would benefit from this).

Reply:
Python floats are implemented as `double` in the Python source code (which is C). So the astropy modeling also uses doubles. I made the above change because the results were actually subtantially different. I think it was my test for writing and reading a `Model` object. It checks `np.testing.assert_allclose(column1, column2, rtol=1e-6, atol=1e-6)`, there were small differences (even with only a 1e-6 tolerance). Not sure why writing/reading would result in these changes, but changing the data type worked.

Reply:
If it writes to ASCII, the number of digits allocated sets the precision.  I know that scipy can use either float or double and will respect your data type.  Can look into it.